src/test/bundles/13-uds-core/uds-bundle.yaml

```diff
@@ -0,0 +1,38 @@
+kind: UDSBundle
+metadata:
+  name: uds-core
+  description: Bundle with Zarf init and UDS Core
+  architecture: amd64
+  version: 0.0.1
+
+packages:
+  - name: uds-k3d
+    repository: ghcr.io/defenseunicorns/packages/uds-k3d
+    # renovate: datasource=github-tags depName=defenseunicorns/uds-k3d versioning=semver
+    ref: 0.4.0
+    overrides:
+      uds-dev-stack:
+        minio:
+          variables:
+            - name: buckets
+              description: "Set Minio Buckets"
+              path: buckets
+            - name: svcaccts
+              description: "Minio Service Accounts"
+              path: svcaccts
+            - name: users
+              description: "Minio Users"
+              path: users
+            - name: policies
+              description: "Minio policies"
+              path: policies
+
+  - name: init
+    repository: oci://ghcr.io/defenseunicorns/packages/init
+    # renovate: datasource=github-tags depName=defenseunicorns/zarf versioning=semver
+    ref: v0.32.1
+
+  - name: core
+    repository: oci://ghcr.io/defenseunicorns/packages/uds/core
+    # renovate: datasource=github-tags depName=defenseunicorns/uds-core versioning=semver
+    ref: 0.13.1-registry1
```

Comment:
I do like the idea of testing against uds-core in general but this would create a weird circle dependency I think? If uds-cli has a breaking change what would the path forward be to cut a release? It seems like you'd have to update uds-core to match the syntax/whatever for the un-released version of uds-cli?

Reply:
That's a really good point that I hadn't thought of. On the one hand, I'd really like to know if we have an unidentified breaking change, but I wouldn't want it to block releases. @justin-o12 what do you think of making this a nightly job instead? (see [example](https://github.com/defenseunicorns/uds-cli/blob/main/.github/workflows/nightly-ghcr.yaml) here)

Reply:
Yeah I guess technically would be a breaking change in zarf probably that would cause the above scenario - didn't think about that... since we're just consuming the zarf package here and the bundle is crafted in this test file. That's probably a pretty unlikely case then?

Reply:
We've switched to deploying the k3d dev bundle, because we do not publish the k3d dev Zarf package to OCI.  So, we're back to your original point that we could have a breaking change.  So, we've moved this workflow to a nightly so it doesn't block releases.